engine/resources/exec.go

```diff
@@ -277,6 +285,7 @@ func (obj *ExecRes) CheckApply(apply bool) (bool, error) {
 		}
 		cmd := exec.Command(cmdName, cmdArgs...)
 		cmd.Dir = obj.IfCwd // run program in pwd if ""
+
```

Comment:
Random nl add? This sort of mistake should be caught by you running `git diff` *before* you submit the patch. Please do =D